components/VM_Arm/src/main.c

```diff
@@ -927,15 +938,38 @@ static int load_linux(vm_t *vm, const char *kernel_name, const char *dtb_name, c
     return 0;
 }
 
-void parse_camkes_linux_attributes(void)
+unsigned int parse_camkes_vm_entry_addr(void)
+{
+    unsigned long addr = strtoul(vm_address_config.kernel_entry_addr, NULL, 0);
```

Comment:
We have to set ENTRY_ADDR_NOT_SET here, right?
```suggestion
    unsigned long addr = strtoul(vm_address_config.kernel_entry_addr, NULL, ENTRY_ADDR_NOT_SET);
```

Reply:
No, ignore that.

Reply:
The third argument of `strtoul` isn't the default value, its a base.

https://www.tutorialspoint.com/c_standard_library/c_function_strtoul.htm

The default being -1 is set here: https://github.com/seL4/camkes-vm/pull/48/files#diff-54f626392a451bea7096290caa1f7d5793d6909bf49da22e0ff19a52c659ca92R78